app/models/post_redirect.rb

```diff
@@ -40,6 +41,14 @@ class PostRedirect < ApplicationRecord
   after_initialize :generate_token
   after_initialize :generate_email_token
 
+  def self.verifier
+    Rails.application.message_verifier(to_s)
+  end
+
+  def self.generate_verifiable_token(user:, value:)
+    verifier.generate(user_id: user.id, nonce: Digest::MD5.hexdigest(value))
```

Comment:
Why am I converting the value to MD5... admins would be able to retrive the old values until the PostRedirect is deleted after two months. Or from the params in the app logs

Reply:
Not really a `nonce`... is there a better name

Reply:
Need to URL encode the token.

Reply:
Confirmed the URL token is encoded correctly